log.txt

```diff
@@ -0,0 +1,40 @@
+236ceeb12883902ce119a42921d5cabd10af7b1a [Structure] Applied php-cs, and enabled in travis
```

Comment:
Maybe let's rename this file to something more meaningful, like `ezpublish-kernel.git.log`.
It's just that if I saw log.txt in the main directory of any repo I'd think that someone commited it by accident :upside_down_face:

Reply:
Not meant to be here. I may have used a brutal git add 🗡